magento2-vulnerable-extensions.csv

```diff
@@ -16,7 +16,7 @@ Magestore_Storelocator,1.0.2,,,https://blog.magestore.com/store-locator-extensio
 Magestore_Storepickup,1.2.1,,,https://blog.magestore.com/store-locator-extension-patch/
 Mirasvit_Blog,1.0.21,,,https://github.com/mirasvit/module-blog
 Mirasvit_Giftr,1.2.23,,https://mirasvit.com/docs/module-gift-registry/current/changelog,https://mirasvit.com/magento-2-extensions/gift-registry.html
-lof_formbuilder,,,https://anothernetsecblog.com/magento-2-extension-security/
-blueformbuilder,,,https://anothernetsecblog.com/magento-2-extension-security/
-vladimirpopov_webforms,2.9.18,,https://anothernetsecblog.com/magento-2-extension-security/
-lof_chatsystem,,,https://anothernetsecblog.com/magento-2-extension-security/
+Lof_Formbuilder,,,https://anothernetsecblog.com/magento-2-extension-security/
```

Comment:
I think you may have the incorrect number of columns?

![image](https://user-images.githubusercontent.com/4132322/83149463-1c7bcc00-a0c8-11ea-8b59-48dc4c9c2ef4.png)

Reply:
Also, the list is currently in alpha order and I think we should keep it that way.

Thanks so much for the contribution, by the way!

Reply:
This is why I shouldn't be on github at 2am :)

Reply:
Hopefully all fixed up now